CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## Pending changes
 
+- [#142](https://github.com/bumble-tech/appyx/issues/142) - **Breaking change**: ActivityIntegrationPoint uses ComponentActivity and explicitly removes entity from Map in onDestroy
```

Comment:
We definitely need to add this to our documentation somewhere.

i.e. `We do not support accessing the integration point within onDestroy`